include/psp2/kernel/modulemgr.h

```diff
@@ -37,23 +37,34 @@ typedef struct SceKernelSegmentInfo
 	SceUInt res;    //!< unused?
 } SceKernelSegmentInfo;
 
+typedef struct {
+	SceUInt32 ptr;  //!< prel31 offset to the start of the function
+	SceUInt32 insn; //!< start of the function (80 B0 B0 B0), 1=cannot be unwound
+} SceKernelModuleExidx;
+
+typedef struct {
+	SceUInt32 insn0;  //!< 
+	SceUInt32 insn1;  //!< 
+	SceUInt32 flags;  //!< often 0
+} SceKernelModuleExtab;
+
 typedef struct
 {
```

Comment:
Mind placing the `{` bracket on the same line as typedef struct and also adding the name?

Reply:
`SceKernelSegmentInfo` and `SceKernelModuleInfo` does not follow these rules.
Should I also update them ?